src/demo-app/stepper/stepper-demo.html

```diff
@@ -0,0 +1,27 @@
+<h2>Horizontal Stepper Demo</h2>
```

Comment:
These are ok for now, but we should eventually add examples that are more like what a developer would actually want to create.